Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -36,17 +36,25 @@ public class AirSinglePipe : MSTSBrakeSystem
         readonly static float OneAtmospherePSI = 14.696f;
         protected float HandbrakePercent;
         protected float CylPressurePSI = 64;
+        protected float CylAirPV;
```

Comment:
Fair point, usually we do put the units in the variable names. Just normally they aren't so chaotic (at least the ideal gas law doesn't care much about units).

Worrying about bar and the more sensible bar * m^3 is a problem for the future. There's some serious cleanup that could be accomplished, and conversion to bar achieved at the same time, but that is well out of scope for today.

Reply:
Would be interesting for the future if we use moles of air instead of P*V (i.e. we would have to divide by the Boltzmann constant and the temperature), or do your formulas explicitly require P*V?

Reply:
I don't strictly need the pressure * volume, that was just the easiest way to measure the "mass" of air in the brake cylinder while maintaining compatibility with the other reservoirs that only track pressure. Everything brake cylinder could be converted to track kgs or moles of air but that would introduce more conversion factors when moving air in and out of the brake cylinder since everything else is PSI and M^3.

For a large-scale refactoring, it would certainly be better to use a standardized (class or struct) air volume for _everything_, not just the brake cylinder, that is manipulated by mass flow instead of the present 'pressure flow' which doesn't make physical sense.

Reply:
Just to clarify the policy on variable names: yes, we prefer units to be included, and we prefer them to be SI or at least metric, but it's not a hard rule.

If the code clarity benefits from no units, simplified units, or non-metric units that's fine.

It should still be clear to those reading the code how to figure out the units.